nv/window.py

```diff
@@ -244,9 +246,12 @@ def window_quit_view(window, **kwargs) -> None:
     # Need to get the setting before quiting the the view because if closing the
     # last view there may not be a view to get the setting from.
     exit_when_quiting_last_window = get_setting(window.active_view(), 'exit_when_quiting_last_window')
+    exit_app_when_quitting_last_window = get_setting(window.active_view(), 'exit_app_when_quiting_last_window')
 
     _close_view(window, **kwargs)
 
+    if len(sublime.windows()) == 1 and exit_app_when_quitting_last_window:
+        sublime.run_command('exit')
```

Comment:
Can you try changing `sublime.run_command('exit')` to `window.run_command('close')` and see if that still works for you. If yes, then it seems the call to `_close_view()` above is not closing your view.  Because below this there is a check `len(window.views()) == 0`.